src/forms/NewStudyForm.js

```diff
@@ -1,351 +1,453 @@
-import React, {useState} from 'react';
+import React, {useState, Fragment} from 'react';
 import PropTypes from 'prop-types';
 import {
   Form,
   Segment,
   Container,
-  Label,
   Button,
   Message,
-  Grid,
   Confirm,
   List,
   Dropdown,
+  Step,
 } from 'semantic-ui-react';
 import {Formik} from 'formik';
 import {workflowOptions} from '../common/enums';
+import {PopupFormField} from '../components/PopupFormField';
 
-const NewStudyForm = ({submitValue, apiErrors}) => {
-  const [confirmOpen, setConfirmOpen] = useState(false);
-  const [workflowType, setSelection] = useState([
-    'bwa_mem',
-    'gatk_haplotypecaller',
-  ]);
+const InfoStep = ({
```

Comment:
Each step component can be broken out into it's own file to help keep this file smaller and make the steps more reusable